pkg/validate/metadata_validate.go

```diff
@@ -0,0 +1,28 @@
+package validate
+
+import "fmt"
+
+func ValidateDefaultChannel(metabundle *MetaBundle) error {
```

Comment:
I would move all validators in a sub-package called `validators/` to split the main code, from the actual validators. 

Also I would have 1 validator per file. So suggesting following directory structure, where `ValidateDefaultChannel` exists in `default_channel.go`:

```bash
pkg
├── utils
│   ├── colors.go
│   └── error.go
└── validate
    ├── print.go
    ├── validate.go
    └── validators
        └── default_channel.go
```